.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+# Scala CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/sample-config/ for more details
+#
+version: 2.1
+
+executors:
+  default:
+    docker:
+    - image: cimg/base:2022.02
+      environment:
+        USER: "circleci"
+        TERM: dumb
+        CI: "true"
+
+jobs:
+  build:
+    executor: default
+    steps:
+    - checkout
+    - run:
+        name: build
+        command: |
+          set -euo pipefail
+
+          echo "It worked!"
```

Comment:
😂

Reply:
For context, the goals of this PR are:

1. Set up the integration with CircleCI for this repo.
2. Set up GitHub branch protection rules to require this CircleCI job to succeed.
3. Provide an entrypoint where the team (you) can put whatever they want to run on their PRs.